velox/common/caching/SsdFile.h

```diff
@@ -378,6 +378,10 @@ class SsdFile {
   // existing checkpoint.
   void logEviction(const std::vector<int32_t>& regions);
 
+  bool isCheckpointEnabled() const {
```

Comment:
NYC: can you also provide method to produce checkpoint and log eviction file paths? Thanks!